### ISSUE TITLE
After FHIR server deployment via portal, schema manager is failing to get the current version and stops db update

### ISSUE THREAD

**Issue by sethuraj** [microsoft/fhir-server#1454]
After the successful deployment of the FHIR server, I tried the schema-manager current action and it gave these error.
It stopped after creating two tables shown in the screen shot. The tables do not have any record.
The subsequent apply -l also failed. 
I tried the same a month ago for my test environment and it just worked fine. 
I left the MS Deploy parameter empty. This is a SQL server version. I am enabling all security settings. I am due to deploy this tonight or tomorrow morning.

schema-manager current -s "https:/

The base schema execution is started.
The base schema execution is completed.
Attempt 1 of 3 to verify if the base schema is synced up with the service.
Attempt 2 of 3 to verify if the base schema is synced up with the service.
Attempt 3 of 3 to verify if the base schema is synced up with the service.
**The current version information could not be fetched from the service. Please try again.**

**FHIR Version?**
R4

**Data provider?**
SQL Server

![image](https://user-images.githubusercontent.com/933249/99204371-4904cc80-276a-11eb-8b80-da81800a49fe.png)

**Comment by CaitlinV39** [comment]
@rbans96 could you review?

**Comment by rbans96** [comment]
@sethuraj  - Was the FHIR server up and running when you hit the current command in the tool? From the error, it seems that the tool could not retrieve the version information that could only happen if the FHIR server is not up.

**Comment by sethuraj** [comment]
Hi @rbans96 
I installed Fhir server and  the app service was in started status. I couldn't try metadata page , I assume because I need the database script run before that. My FHIR database is empty.
I am going to reinstall this tonight and will let you know.
Thanks
Jay

**Comment by sethuraj** [comment]
Hi @rbans96 
I reinstalled. WebApp service seems to be running, but I can't access my metadata by url. So not sure whether the server is up and running or not.  The root appservice url is also returning "cant be reached message"

When the FHIR Server is deployed , FHIR database does not have any tables.
![image](https://user-images.githubusercontent.com/933249/99838822-90021180-2b1e-11eb-8e91-97dfc3f48492.png)
schema_manager is failing, So I used the 4.sql to create the database. 
restarted the app service still no luck.

Few weeks ago I did the same for my test environment and I didn't have any issue.

**Comment by rbans96** [comment]
I see. It seems you are deploying the app service code prior to this release tag - https://github.com/microsoft/fhir-server/releases/tag/build%2F20201111-4. If yes, there was a known issue that the app service does not starts without the schema initialization that could be the reason metadata endpoint could not be accessed.

Just executing the 4.sql would not work. Since migrating the schema by schema migration tool or via automatic schema upgrades executes baseSchema.sql then the latest versioned script and completes the schema version.

So if you deploy from the tag https://github.com/microsoft/fhir-server/releases/tag/build%2F20201111-4 or any release after that would resolve this issue for you. But if for any reason, want to deploy the old code, then please update the configuration setting sqlSchemaAutomaticUpdatesEnabled to true which will automatically initialize the schema for you while appservice deployment.

**Comment by sethuraj** [comment]
@rbans96 
I  have been using the same resource group and the the app service plan that I created separate myself (windows).

This time I created a new resource group. Did the app registration using Powershell commands
New ASP was given in the template to create it during the deployment and its just created everything.

My schema-Manager I ran for version 4 and then I ran the latest.
This time I left the MSDeploy empty to take the default. Will that give me the latest stable version?

![image](https://user-images.githubusercontent.com/933249/99850423-3277c000-2b32-11eb-8870-803def3046fa.png)

Everything seems good now. Metadata url is working.
I am going to test further with my authorization key.
